src/connectors/Receiver.sol

```diff
@@ -5,4 +5,5 @@ import {UpgradeableBase} from "./UpgradeableBase.sol";
 
 contract Receiver is UpgradeableBase {
```

Comment:
Is it somehow better than to add receive method directly?
Why are you inheriting UpgradeableBase? You are also inheriting it in other contracts that are using this receiver. Maybe it isn't causing any problems, but looks a little bit messy

Reply:
Yes, Receiver was supposed to be just a simple reuse of `receive()`. Fixed in the last commit.